src/key.js

```diff
@@ -395,13 +428,11 @@ function namespaceValue (ns) {
 /**
  * Flatten array of arrays (only one level)
  *
- * @param {Array<Array>} arr
- * @returns {*}
+ * @param {Array<any>} arr
```

Comment:
Looks like this should be a generic function insetad:

```suggestion
 * @template T
 * @param {Array<T|T[]>} arr
 * @returns {T[]}
```